gravitee-apim-console-webui/src/management/api/apis-routing.module.ts

```diff
@@ -853,16 +853,6 @@ const apisRoutes: Routes = [
             },
             component: ApiDocumentationV4EditPageComponent,
           },
-          {
-            path: 'default-pages/homepage/new',
```

Comment:
I'm not a huge fan of removing "homepage" from the path because we will run into a problem once Terms and Conditions are added to the page.

The cleanest solution would be to keep the navigation as it was, and then split the component into a dummy vs. smart component:
- Create a new dummy component for rendering the stepper + stepper-header
- Create a new smart component that would add the title (ex. "Default Pages" + subtitle) and control what to do on the actions: "Save and Publish", "Save", "Cancel", "Exit without saving".